### DISCUSSION TITLE
How to increase  the storege

### DISCUSSION THREAD

**Discussion by lslvxy** [Genymobile/feedback#160] [Genymotion Desktop] [answered]
total storege only 32GB, system used about 20 GB, There is not enough remaining storage space.

**Comment by Dannflower** [comment, marked as answer]
After creating a virtual device, you'll need to go find that device's image and increase its maximum size. Once you've done that, you'll need to increase the size of the `data` partition within that image.

I made a guide for my own reference to do this on Linux, but maybe it'll be helpful to you as well as a general guide: https://github.com/Dannflower/android-gaming-on-linux?tab=readme-ov-file#increasing-device-storage

**Reply by lslvxy** [reply]
Nice work, successful increased my qcow2 img

**Reply by PumpkinXD** [reply]
hmmm I wonder how to increase virtualbox's image...

there's `data.vmdk` and `Snapshots/{uuid}.vmdk` and idk which I should resize (and their partation seems weird
<img width="577" height="556" alt="图片" src="https://github.com/user-attachments/assets/91165eeb-82fb-4228-ade8-881acdb6d4bc" />

**Reply by jmaneyrol69** [reply]
@PumpkinXD using VirtualBox on Linux for Genymotion is deprecated. Default and recommended hypervisor on Linux is now QEMU (see https://docs.genymotion.com/desktop/Get_started/013_Linux_install/#hypervisor).

**Reply by PumpkinXD** [reply]
> @PumpkinXD using VirtualBox on Linux for Genymotion is deprecated. Default and recommended hypervisor on Linux is now QEMU (see https://docs.genymotion.com/desktop/Get_started/013_Linux_install/#hypervisor).

tried qemu before and I selected back, but seems no hardware acceleration(while playing arknights...)

qemu installed via dnf(my distro is fedora)

guest is android 11, `amazon fire hd 10`

**Reply by jmaneyrol69** [reply]
@PumpkinXD you do not need to install QEMU: it is embedded with Genymotion. It simply uses KVM which should be installed by default on Fedora Workstation. See https://docs.genymotion.com/desktop/Get_started/013_Linux_install/. Also, we have not encountered any hardware acceleration issues on our test machines (Ubuntu & Fedora Workstation).
By the way, are you trying to run Genymotion in a VM? If so, it will not work: Genymotion does not support nested virtualization.

**Reply by PumpkinXD** [reply]
no, my Fedora is on bare metal(and I got a nvidia 2060 card... official driver from nvidia rather than nouveau)

whilst genymotaion running, plasma-systemmonitor says 0% gpu usage(_maybe this is inaccurate since it also shows 0 when I launching minecraft_) and arknights in the guest got fps spikes a lot and sometimes kills the guest...

**Reply by jmaneyrol69** [reply]
@PumpkinXD Servers (Bare metal or others) are not supported either. This is clearly stated in the on-line documentation (https://docs.genymotion.com/desktop/Get_started/Requirements/) and the FAQ (https://support.genymotion.com/hc/en-us/articles/360000290798-Can-Genymotion-Desktop-run-in-a-server-Is-there-a-headless-mode). Only Desktop machines are supported.

**Reply by PumpkinXD** [reply]
> @PumpkinXD Servers (Bare metal or others) are not supported either. This is clearly stated in the on-line documentation (https://docs.genymotion.com/desktop/Get_started/Requirements/) and the FAQ (https://support.genymotion.com/hc/en-us/articles/360000290798-Can-Genymotion-Desktop-run-in-a-server-Is-there-a-headless-mode). Only Desktop machines are supported.

uh sorry, my previous reply might be misleading, my machine is an alienware laptop("bare-metal" just mean I'm not trying installing inside a vm guest)
<img width="916" height="638" alt="图片" src="https://github.com/user-attachments/assets/1d9b2a3d-ffc2-4e20-b3ab-0275ac4be84e" />

**Reply by jmaneyrol69** [reply]
@PumpkinXD thank you for the clarification. As said previously, do not use VirtualBox - this hypervisor is deprecated and only kept for compatibility purposes. The solution provided by @Dannflower is the only known working solution so far. Have you tried following his instructions?
